environments/templates/nublado-application.yaml

```diff
@@ -7,6 +7,14 @@ spec:
   finalizers:
     - "kubernetes"
 ---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "fileservers"
+spec:
+  finalizers:
+    - "kubernetes"
+---
```

Comment:
We want the created fileservers to be visible in Argo CD, so this should use the same approach as we use for the nublado-users application (but also create a namespace). So there needs to be another Argo CD application that we enable if the file server is enabled.

I've been thinking that it may be better to create those Argo CD applications directly in the nublado Helm chart rather than creating them in environments, since they're really implementation details of nublado and Argo CD supports that app of app construction. So ideally you'd also move the creation of the nublado-users app into nublado along with this.